python/triqs/operators/util/U_matrix.py

```diff
@@ -131,43 +135,88 @@ def reduce_4index_to_2index(U_4index):
 
     return U, Uprime
 
-# Construct the 2-index matrices for the density-density form
-def U_matrix_kanamori(n_orb, U_int, J_hund):
+def U_matrix_kanamori(n_orb, U_int, J_hund, Up_int=None, full_Uijkl=False, Jc_hund=None):
     r"""
-    Calculate the Kanamori U and Uprime matrices.
+    Calculate the Kanamori two-index interaction matrix for parallel spins:
+
+    .. math:: U_{m m'}^{\sigma \sigma} \equiv U_{m m' m m'} - J_{m m'}
+
+    with:
+
+    .. math:: J_{m m'} \equiv U_{m m' m' m} ,
+
+    and the two-index interaction matrix for anti-parallel spins:
+
+    .. math:: U_{m m'}^{\sigma \bar{\sigma}} \equiv U_{m m' m m'}
+
+    If full_Uijkl=True is specified instead the full four index 
```

Comment:
It would be nice to have a unit test that compares two Hamiltonians constructed from `U`/`Uprime` and from `Uijkl`.

Reply:
addressed in e6725183